KarhooUISDK/UIComponents/AddressBarMVP/PrebookFieldMVP/KarhooPrebookFieldView.swift

```diff
@@ -37,44 +48,109 @@ public final class KarhooPrebookFieldView: UIView {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    // MARK: - Setup
+    
+    private func setupHierarchy() {
+        addSubview(stackContainer)
+        stackContainer.addArrangedSubview(dateTimeView)
+        addSubview(dateTimeStackContainer)
+        dateTimeStackContainer.addArrangedSubview(timeLabel)
+        dateTimeStackContainer.addArrangedSubview(dateLabel)
+        dateTimeView.addSubview(closeButton)
+        stackContainer.addArrangedSubview(dateTimeView)
+        stackContainer.addArrangedSubview(prebookButton)
+    }
     
     private func setUpView() {
         translatesAutoresizingMaskIntoConstraints = false
+        setupStackView()
+        setupDateTimeView()
+        setupDateTimeStackView()
+        setupTimeLabel()
+        setupDateLabel()
+        setupCloseButton()
+        setupPrebookButton()
+        setupHierarchy()
+        setUpConstraints()
+    }
+    
+    private func setUpConstraints() {
```

Comment:
Try using the `myView.**anchor**(...)` method wherever possible instead of adding constraints and activating them manually